engine/src/main/java/org/terasology/math/Vector3fUtil.java

```diff
@@ -56,9 +56,11 @@ public static Vector3f getPerpendicularComponent(Vector3f direction, Vector3f no
     }
 
     public static Vector3f safeNormalize(Vector3f v, Vector3f out) {
+        final float EPSILON = 0.000001f;
```

Comment:
I can make this constant global, but it wouldn't make sense because it isn't used anywhere else. If we can find a solution for consideration 4, we wouldn't need this constant at all.

Reply:
Actually, if you do a project-wide search for the string "epsil" you'll find a few other places where an epsilon is used to name a small number.

Reply:
True. Making a single `EPSILON` constant which is used everywhere is a good a idea, but is out of scope for this PR (at best, loosely related with consideration 4). The only reason I made this change was to remove the usage of  `BulletGlobals.SIMD_EPSILON` (in the TeraBullet package) in the `safeNormalize` method.

Reply:
Ok, the out-of-scope is a good argument. Can you please write a brief issue on the idea of having a global epsilon constant, so that the concept doesn't get lost?

Reply:
Definitely. Will make one soon enough. I like the idea of centralizing the global epsilon, as part of the physics engine system or otherwise.